src/controllers/orderController.js

```diff
@@ -162,4 +162,51 @@ const deleteOrder = async (req, res) => {
   */
 };
 
-module.exports = { createOrder, getAllOrders, getSingleOrder, deleteOrder };
+const handleStripePayment = async (req) => {
+  try {
+    // Receive Stripe Signature
+    const stripeSignature = req.headers['stripe-signature'];
+
+    //  Construct the Webhook Event
+    const event = stripe.webhooks.constructEvent(
+      req.rawBody, // Raw text body payload received from Stripe
```

Comment:
Since we run the `express.raw` middleware in `expressServer.js` for this endpoint, that middleware puts the raw bytes body into the `req.body`  property, so we can access it there (I think here `req.rawBody` may just be undefined, but I haven't checked it myself with console logging)

```suggestion
      req.body, // Raw text body payload received from Stripe
```